docs/source/conf.py

```diff
@@ -27,16 +26,8 @@
 author = 'ReenigneArcher'
 
 # The full version, including alpha/beta/rc tags
-with open(os.path.join(root_dir, 'CMakeLists.txt'), 'r') as f:
-    version = re.search(r"project\(Sunshine VERSION ((\d+)\.(\d+)\.(\d+))", str(f.read())).group(1)
-"""
-To use cmake method for obtaining version instead of regex,
-1. Within CMakeLists.txt add the following line without backticks:
-   ``configure_file(docs/source/conf.py.in "${CMAKE_CURRENT_SOURCE_DIR}/docs/source/conf.py" @ONLY)``
-2. Rename this file to ``conf.py.in``
-3. Uncomment the next line
-"""
-# version = '@PROJECT_VERSION@'  # use this for cmake configure_file method
+# https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_VERSION
+version = os.getenv('READTHEDOCS_VERSION', 'dirty')
```

Comment:
The version isn't actually used by readthedocs build, and this simplifies the tests.